inc/Image-sizes.php

```diff
@@ -8,15 +8,6 @@
  // add_image_size
 });
 
-// The srcset provided by WP is... weird.
-add_filter("wp_calculate_image_sizes", function ($sizes) {
```

Comment:
IIRC, the reasoning for this code block is that when someone wanted a full width image in the post content, the default srcset went and fucked it all up.

Reply:
Okay, so after testing this more, it seems that this filter is also related to the `$GLOBALS["content_width"]` -variable. Increasing this global value seems to start adding the different image sizes into the markup, so it also makes this filter useless in a sense that it doesn't apply anymore? The way these are intertwined is really weird.

I'd be inclined to just keep this simple and remove this filter because the end-user will be inserting the content to `post_content`, so I'd say we need to keep the site "safe" from the original images by adding the srcset-attribute always when possible. I guess I'd need to see a real-world use-case where the srcset-attribute would do some harm 😄